app/views/articles/_badges_area.html.erb

```diff
@@ -54,4 +54,24 @@
       <% end %>
     </div>
   </div>
+<% else %>
+  <style>
+    .widget-body {
+      margin: 0 auto;
+      padding: 20px;
+    }
+
+    @media screen and (min-width: 1024px) {
+      .no-badges-text {
+        display: none;
+      }
+    }
+  </style>
+  <div style="overflow: visible;padding-top:20px;">
+    <div class="widget-body">
+      <p class="no-badges-text">
+        Eat and than sleep on your face annoy the old grumpy cat, start a fight and then retreat to wash when i lose and i shall purr myself to sleep or run around the house at 4 in the morning
```

Comment:
to be nitpicky: `i` should be capitalized to `I`, correct?

Thanks for this @cesc1989!

Reply:
Agreed, @rhymes . Doing the change:)!